<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>LM Roman Slanted 10</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="ArduinoStatus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>391</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
     </font>
    </property>
    <property name="text">
     <string>Arduino Status: Not Connected</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>23</y>
      <width>221</width>
      <height>555</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Surge (X)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_X">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-30</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Sway (Y)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_Y">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-30</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Heave (Z)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_Z">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-30</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Roll (phi)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_PHI">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-45</number>
       </property>
       <property name="maximum">
        <number>45</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Pitch (theta)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_THETA">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-45</number>
       </property>
       <property name="maximum">
        <number>45</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>Yaw (psi)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_PSI">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="minimum">
        <number>-45</number>
       </property>
       <property name="maximum">
        <number>45</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
        </font>
       </property>
       <property name="text">
        <string>ABORT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
